reusable-workflows/python_linter.yml

```diff
@@ -76,18 +76,22 @@ jobs:
               if: ${{ inputs.use_black }}
               run: |
                 black ${{ inputs.directory_to_lint }} --check --diff
+              working-directory: ${{ inputs.working_directory }}
 
             - name: Lint with flake8 (PEP8 enforcer + linter)
               if: ${{ inputs.use_flake8 }}
               run: |
                 flake8 ${{ inputs.directory_to_lint }}  --show-source
+              working-directory: ${{ inputs.working_directory }}
 
             - name: isort
               if: ${{ inputs.use_isort }}
               run: |
                 isort ${{ inputs.directory_to_lint }} --profile black --filter-files --check-only --diff
+              working-directory: ${{ inputs.working_directory }}
 
             - name: pylint
               if: ${{ inputs.use_pylint }}
               run: |
                   pylint ${{ inputs.directory_to_lint }}
+              working-directory: ${{ inputs.working_directory }}
```

Comment:
Other than having to fix the command line ( ${{ inputs.directory_to_lint }} ), using the `working directory` means that we are forcing users to have all the files that they want in a single parent directory (thing that is wrong, for example, for the service portal). Having the directory (or directories, using the same trick that we have in the `strategy` ) in the command line means that we can actually manage this case